examples/config/telemetry/otel.go

```diff
@@ -0,0 +1,91 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package telemetry
+
+import (
+	"context"
+	"errors"
+	"os"
+	"sync"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"go.opentelemetry.io/contrib/bridges/otelzap"
+	config "go.opentelemetry.io/contrib/config/v0.3.0"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/log/global"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/trace"
+)
+
+var (
+	// Ensure that the logger is only created once.
+	loggerOnce sync.Once
+
+	// By default, use a no-op logger.
+	logger = zap.NewNop()
```

Comment:
Can we show how to use `slog` instead? It is a part of the standard library and I think we should promote that use.

Reply:
I updated the PR to include this, but it wasn't immediately clear to me how to dual write to the console and to an OTLP endpoint in a similar way to how zap does. I can certainly add a console exporter to the otel.yaml config, but that's not useful at all for production purposes.